frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -518,6 +518,40 @@ export const dashboardLogic = kea<dashboardLogicType>({
         ],
     }),
     selectors: () => ({
+        asDashboardTemplate: [
+            (s) => [s.allItems],
+            (dashboard: DashboardType): string => {
+                return dashboard
+                    ? JSON.stringify({
```

Comment:
`JSON.stringify(..., undefined, 4)` for formatting with 4 spaces of indentation ;)